README.md

```diff
@@ -33,7 +33,11 @@ Many of the examples here are directly from the Project Jigsaw: Module System Qu
          `[destination]` usually `/usr/lib/...` or related folder
     
     - MacOSX 
+      
        #### Manual way
+       
```

Comment:
Any reasons for these spaces, might not have any effect on the markdown visually